src/QPSReader.jl

```diff
@@ -4,4 +4,29 @@ export QPSData, readqps
 
 include("readqps.jl")
 
+if VERSION ≥ v"1.3"
+  using Pkg.Artifacts
+
+  export netlib_path, mm_path
+
+  # for some reason, artifact"netliblp" doesn't work...?!
+  artifact_toml = joinpath(@__DIR__, "..", "Artifacts.toml")
+  ensure_artifact_installed("netliblp", artifact_toml)
+  netliblp_hash = artifact_hash("netliblp", artifact_toml)
+  @assert artifact_exists(netliblp_hash)
+
+  @doc "Path to the Netlib linear optimization test set."
+  const netlib_path = joinpath(artifact_path(netliblp_hash), "optrove-netlib-lp-f83996fca937")
+
+  ensure_artifact_installed("marosmeszaros", artifact_toml)
+  mm_hash = artifact_hash("marosmeszaros", artifact_toml)
+  @assert artifact_exists(mm_hash)
+
+  @doc "Path to the Maros-Meszaros quadratic optimization test set."
+  const mm_path = joinpath(artifact_path(mm_hash), "optrove-maros-meszaros-9adfb5707b1e")
+
+  # const netlib_path = joinpath(artifact"netliblp", "optrove-netlib-lp-f83996fca937")
+  # const mm_path = joinpath(artifact"marosmeszaros", "optrove-maros-meszaros-9adfb5707b1e")
```

Comment:
Maybe just remove those two lines?

Reply:
In principle, only those two lines are necessary (they are supposed to do the same as the stuff above). But for some reason, they return an error. I asked on Slack to see if anyone has an idea.